Documentation/CRDs/specification.md

```diff
@@ -8027,7 +8027,8 @@ string
 </em>
 </td>
 <td>
-<p>BalancerMode sets the <code>balancer</code> module with different modes like <code>upmap</code>, <code>crush-compact</code> etc</p>
+<p>BalancerMode sets the <code>balancer</code> module with different modes like <code>upmap</code>, <code>crush-compact</code> etc.
+Note: Ceph cluster should be on <code>Reef</code> or higher in order to use <code>read</code> and <code>upmap-read</code> modes.</p>
```

Comment:
For downstream, the customers will be upgrading from 4.16 (Reef) to 4.17(Squid). So we won't need any specific checks if customer is already on Reef for downstream. 

For upstream, the above warning should suffice for the users, IMO.  

@travisn